### DISCUSSION TITLE
Deploying via Github Actions

### DISCUSSION THREAD

**Discussion by daltonrooney** [thegetty/quire#987] [Development]
Hi there, I'm trying to deploy via Github Actions, using an Ubuntu VM. We deploy a lot of static sites this way and generally don't have a problem. Quire doesn't seem to want to build, though, and isn't giving me any details about why it's failing. 

I ran `quire build --debug` and compared it with a local debug log and it seems like it's stopping right before the line 
```
[quire] INFO	 Figures                        Processing complete
```

It never gets to 
```
Eleventy:EleventyFiles Searching for: [ './content/**/*.{11ty.js,11ty.cjs,hbs,liquid,md,njk,webc}' ]
```
and just dies silently.

Has anyone else tried deploying via Github Actions with success? Or Ubuntu in general?

**Comment by Erin-Cecele** [comment]
@chrisdaaz I believe you are an Ubuntu user. Do you have any guidance or advice for @daltonrooney? 

On a separate note, @daltonrooney, we would love to learn more about the project you are working on!

**Reply by daltonrooney** [reply]
@Erin-Cecele I'm working with the Mingei International Museum here in San Diego on their Blue Gold exhibition. We manage their main website and set up a virtual host on their web server for the digital publication. We're able to build locally and sync the files but are hoping to automate it so the Mingei staff can make updates without requiring an extra deploy step.

On a whim I changed the VM to `macos-latest` (I actually didn't know that was an option on GitHub Actions!) and I think it's further along. I'll report back here if I'm able to get everything working.

**Reply by Erin-Cecele** [reply]
Ah yes! I've been in close contact with Kristi. We are very excited for the publication release in the new year.

Glad to hear you may be making some progress. Please keep us updated!

**Comment by chrisdaaz** [comment]
I've deployed to GitHub Pages with GitHub Actions here:https://github.com/chrisdaaz/recasting-antiquity/blob/main/.github/workflows/main.yml

This repo is strictly the content and I replace all the starter files in Quire from a fresh Quire install with my repo's content at build time. This project is almost a year old, however.

**Reply by daltonrooney** [reply]
@chrisdaaz Thanks for your reply. Switching from Ubuntu to MacOs fixed the issue and we're now able to run the rest of the script successfully.
